packages/papirus-icon-theme/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/PapirusDevelopmentTeam/papirus-icon-theme
+TERMUX_PKG_DESCRIPTION="Papirus open source icon theme for Linux
+ Papirus is a SVG-based icon theme, drawing inspiration from Material Design
+ and flat design.
+ .
+ This package contains the following icon themes:
```

Comment:
Please Keep the description in a single line..
Apt doesn't support multiline description very well